lib/spack/spack/build_systems/cmake.py

```diff
@@ -29,6 +29,7 @@
 
 import llnl.util.tty as tty
 import spack.build_environment
+import spack.directives
```

Comment:
Just to track what we are importing from where. As we are in core I tried to avoid:
```python
from XXX import *
```
but if you think it's more readable, I have no objection changing it.